CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ option(BUILD_COVER "Generate test coverage" OFF)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 include(ClangFormat)
+if (WIN32)
+        set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP ON)
+    include(InstallRequiredSystemLibraries)
+endif()
```

Comment:
They will be provided, but it will be done manually by me (at least now).
It is not needed to install them now, because all dlls won't be installed anyway (Qt dlls, pnglib dlls, SFML dlls, etc). It will be confusing to have some random dlls installed, and other not.
The proper deployment on Windows is a separate task.